docs/admin/system-information.rst

```diff
@@ -834,9 +834,16 @@ Recovery expected data nodes
 
 The check for the
 :ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting checks
-that the number of data nodes that should be waited for the immediate
-cluster state :ref:`recovery <gloss-shard-recovery>`. This value must be equal
-to the maximum number of data nodes in the cluster.
+if the setting match the actual number of data nodes present in the cluster.
+The value of this settings is checked against 
+:ref:`gateway.recover_after_data_nodes
+<gateway.recover_after_data_nodes>` (or 
+:ref:`gateway.recover_after_nodes <gateway.recover_after_nodes>`
+) during startup to decide if the cluster can proceed with 
```

Comment:
I think the closing bracket needs to be on the previous line, otherwise, this renders an additional space:
<img width="831" alt="Screenshot 2022-12-30 at 14 31 57" src="https://user-images.githubusercontent.com/218003/210075594-0014bd86-817f-4c2e-9246-8e83d03b2de0.png">